### DISCUSSION TITLE
New translation in Romany (Southern Greece)

### DISCUSSION THREAD

**Discussion by jimrp** [sircharlo/meeting-media-manager#2159] [Translations]
### What language do you speak?

I would like to help translate M³ into a language I speak, Romany (Southern Greece) (RMG/rmn-x-rmg).

**Comment by sircharlo** [comment]
Thank you so much for your willing spirit!
If you would like to volunteer to become a M³ language moderator for a language you speak, please [sign into Crowdin](https://crowdin.com/project/meeting-media-manager) using your GitHub account, and then let me know once that is done.

As language moderator, you will get notified of changes to the English original strings, have a nice and easy to use interface to review existing as well as untranslated strings in your language, and submit new translations to the repo easily.

**Reply by jimrp** [reply]
Thank you, ok I signed up.

**Reply by sircharlo** [reply]
Ok! I've invited you, you should see an invitation to translate this language in Crowdin.
